<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Round Gauge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="minimum">
       <double>-50.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Update Gauge</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,0">
      <item>
       <widget class="QPushButton" name="outerRingColorButton">
        <property name="text">
         <string>Outer Ring Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="valueColorButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Value Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="glowRingColorButton">
        <property name="text">
         <string>Glow Ring Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fontColorButton">
        <property name="text">
         <string>Font Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="glowRingEnabledCheck">
        <property name="text">
         <string>Glow Ring Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Value:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
